GenerateCLP/GenerateCLP.cxx

```diff
@@ -560,6 +560,39 @@ void GenerateDeSerialization( std::ostream & sout,
             | "          deserializedVectorFlaggedArgs.push_back(\"" + flag + "\");"
```

Comment:
`When an multiple argument is present ...`

->

`Argument described using "multiple=true" are found in both JSON and in the command line, similarly to other arguments, the values of the JSON are ignored.`